rasa/cli/data.py

```diff
@@ -235,6 +262,54 @@ def split_nlu_data(args: argparse.Namespace) -> None:
     telemetry.track_data_split(args.training_fraction, "nlu")
 
 
+def augment_nlu_data(args: argparse.Namespace) -> None:
+    """Performs data augmentation for NLU.
+
+    Data augementation works by determining intents with low performance or little data from the existing training data
+    and the generated paraphrases are used as additional training data for these intents.
+
+    Subsequently, two models with augmented data are trained and then evaluated on the supplied test data.
+
+    The module outputs a report highlighting how data augmentation has changed classification performance.
+    
+    Args:
+        args: Command-line arguments.
+    """
+    rasa.shared.utils.common.mark_as_experimental_feature(
+        "Data augmentation by paraphrasing for NLU"
+    )
+
+    # Collect inputs
```

Comment:
We should add some logging statements to let the user know what's happening here - I suggested some [here](https://github.com/RasaHQ/rasa/pull/7992/files#diff-3f4df023f11d87183849308c609e9a29b5a878aed1ca646ef9be1681036376d1R608-R609) but feel free to add more.